cpp/modmesh/buffer/SimpleArray.hpp

```diff
@@ -307,6 +309,26 @@ class SimpleArray
         }
     }
 
+    explicit SimpleArray(small_vector<size_t> const & shape, small_vector<size_t> const & stride, std::shared_ptr<buffer_type> const & buffer)
+        : SimpleArray(buffer)
+    {
+        if (buffer)
+        {
+            m_shape = shape;
+            m_stride = stride;
+            const size_t nbytes = std::accumulate(m_shape.begin(),
+                                                  m_shape.end(),
+                                                  static_cast<size_t>(1),
+                                                  std::multiplies<size_t>()) *
+                                  ITEMSIZE;
```

Comment:
Would the expression look nicer by placing `ITEMSIZE` before `std::accumulate`?

Reply:
That's a good idea.